docs/platform/one-click/deploying-docker-with-one-click-apps/index.md

```diff
@@ -0,0 +1,97 @@
+ - '[Docker Hub](https://www.docker.com/products/docker-hub)'
+---
+
+## Docker One-Click App
+
+Docker is a tool that enables you to create, deploy, and manage lightweight, stand-alone packages that contain everything needed to run an application (code, libraries, runtime, system settings, and dependencies). These packages are called containers.
+
+Each container is deployed with its own CPU, memory, block I/O, and network resources, all without having to depend upon an individual kernel and operating system. While it may be easiest to compare Docker and virtual machines, they differ in the way they share or dedicate resources.
+
+### Deploy a Docker One-Click App
+
+{{< content "deploy-one-click-apps">}}
+
+### Linode Options
+
+Docker doesn't ask you to provide any app specific configurations. Provide configurations for your Linode server:
+
+| **Configuration** | **Description** |
+|:--------------|:------------|
+| **Select an Image** | Debian 9 is currently the only image supported by Docker One-Click Apps, and it is pre-selected on the Linode creation page. *Required*. |
+| **Region** | The region where you would like your Linode to reside. In general, it's best to choose a location that's closest to you. For more information on choosing a DC, review the [How to Choose a Data Center](/docs/platform/how-to-choose-a-data-center) guide. You can also generate [MTR reports](/docs/networking/diagnostics/diagnosing-network-issues-with-mtr/) for a deeper look at the network routes between you and each of our data centers. *Required*. |
+| **Linode Plan** | Your Linode's [hardware resources](/docs/platform/how-to-choose-a-linode-plan/#hardware-resource-definitions). Docker can be supported on any size Linode, but we suggest you choose a Linode plan that reflects how many resources you plan on using. For small applications, a 1GB Nanode is sufficient. If you decide that you need more or fewer hardware resources after you deploy your app, you can always [resize your Linode](/docs/platform/disk-images/resizing-a-linode/) to a different plan. *Required*. |
+| **Linode Label** | The name for your Linode, which must be unique between all of the Linodes on your account. This name will be how you identify your server in the Cloud Manager’s Dashboard. *Required*. |
+| **Root Password** | The primary administrative password for your Linode instance. This password must be provided when you log in to your Linode via SSH. It must be at least 6 characters long and contain characters from two of the following categories: lowercase and uppercase case letters, numbers, and punctuation characters. Your root password can be used to perform any action on your server, so make it long, complex, and unique. *Required*. |
+
+When you've provided all required Linode Options, click on the **Create** button. **Your Docker app will complete installation anywhere between 2-5 minutes after your Linode has finished provisioning**.
+
+## Getting Started after Deployment
+
```

Comment:
Maybe here it can be specified that the following instructions are provided just as an example on how to use Docker, not necessarily steps you must absolutely take to use Docker.

Reply:
added a couple of sentences here, let me know what you think.